### PR TITLE
Fixed err to be error

### PR DESCRIPTION
**Pull request by vavalos5**
#### What does this PR do?
Spelled out error to make it more clear to users what "err" means

### CODE REVIEW

**Review by britt6612 — APPROVED** [review]
good catch

File: aries-site/src/pages/components/radiobuttongroup.mdx [hunk 1]
```diff
@@ -19,7 +19,7 @@ import {
 
 ### Best Practices
 
-- Keep labels concise and simple. Avoid lengthy descriptions, however err on clarity over brevity.
+- Keep labels concise and simple. Avoid lengthy descriptions, however error on clarity over brevity.
```
Comment (halocline) [inline aries-site/src/pages/components/radiobuttongroup.mdx:22]: Err is a verb (i.e. to make a mistake) and is used correctly here. Error is a noun and is incorrect here.

```suggestion
- Keep labels concise and simple. Avoid lengthy descriptions, however err on clarity over brevity.
```

**Review by halocline — CHANGES_REQUESTED** [review]

File: aries-site/src/pages/components/radiobuttongroup.mdx (hunk 1 above)
Comment (halocline) [reply aries-site/src/pages/components/radiobuttongroup.mdx:22]: https://www.merriam-webster.com/dictionary/err

https://www.beedictionary.com/common-errors/error_vs_err

Comment (britt6612) [reply aries-site/src/pages/components/radiobuttongroup.mdx:22]: thanks for this I never knew that! I also thought it was a typo

Comment (vavalos5) [reply aries-site/src/pages/components/radiobuttongroup.mdx:22]: Thanks for the clarification @halocline Learned something new

**Pull request closed without merging**